BuildingAndTestingInBunsen.md

```diff
@@ -0,0 +1,160 @@
+# Building and Testing the VDO software in a Bunsen Environment
+
+This guide is intended to help get you started with developing, building, and testing your contributions to VDO in a Bunsen environment.
+
+# Prerequisites
+
+It is expected that you have satisfied the following criteria before getting started:
+
+* [Set up a Bunsen Environment](Getting_Started.md)
+
+# Expectations of this Guide
+
+This guide is currently intended to provide enough information to get minimally functional.  A more detailed guide will be generated later, but in the meantime this document should help you get started if you're interested in contributing to the VDO projects.
```

Comment:
Line 13: This guide is intended to provide enough information to get you started in contributing to the VDO projects. A more detailed guide is being generated and will be available at a later date.

Reply:
Adapted and available in the latest revision of this PR.